### ISSUE TITLE
Not respecting the -ExcludeCharacters parameter

### ISSUE THREAD

**Issue by aaroncorgan** [rhymeswithmogul/PSPasswordGenerator#9]
Hello, unsure which of the parameters im using is causing this issue, or perhaps its the combination of them all, but no matter what I do, I cannot get passwords to generate using a wordlist that respects the excludecharacters parameters. Im simply trying to make the separating characters between words as simple as possible - a `includecharacters` whitelist would actually be better, i would simply just separate each word with a '.' period. 

Ive tried only a single exclusion, many exclusions, wrapping them in single quotes, not wrapping them in anything (and ensuring im not using any characters that would cause issues). Im using the -AsPlainText parameter and the [orchard street medium list ](https://github.com/sts10/orchard-street-wordlists/blob/main/lists/orchard-street-medium.txt) as my word list, saved as a .txt file, if that matters. Heres some example output to show the issue:

```powershell
PowerShell 7.4.6
Loading personal and system profiles took 4066ms.
PS C:\PowerShell> install-psresource pspasswordgenerator

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its Trusted value by
running the Set-PSResourceRepository cmdlet. Are you sure you want to install the PSResource from 'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): a
PS C:\PowerShell> import-module PSPasswordGenerator
PS C:\PowerShell> get-randomPassword -WordList "C:\PowerShell\wordlist.txt" -Words 8 -ExcludeCharacters '!@#$%^&*(){}+|\<>?123456789~/' -AsPlainText
residue!pioneering(photos3Message[Decisive=plants`judge;gastric
```

**Comment by rhymeswithmogul** [comment]
No, that looks like a bug. Let me dig through my code and see what's up. Things are a little hectic right now, so this may take some time.
